types/adobe__pdfservices-node-sdk/operation/insert-pages-operation.d.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * An operation that can be used to insert pages of multiple PDF files into a base PDF file.
+ * <p>
+ * For more complex use cases, refer the {@link CombineFilesOperation}
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        InsertPages = PDFServicesSdk.InsertPages,
+ *        insertPagesOperation = InsertPages.Operation.createNew();
+ *
+ *  insertPagesOperation.setBaseInput(PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/insertPagesOperationBaseInput.pdf',
+ *  	InsertPages.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/insert-pages-operation.d.ts;
  Line: 17;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.